### PR TITLE
fix for unsafe validator

### PR DESCRIPTION
**Pull request by mdmunir**
Closes #8145 #8139 #11153

### CODE REVIEW

File: framework/validators/Validator.php [hunk 1]
```diff
@@ -231,9 +231,22 @@ public function init()
     public function validateAttributes($model, $attributes = null)
     {
         if (is_array($attributes)) {
-            $attributes = array_intersect($this->attributes, $attributes);
+            $attrs = [];
```
Comment (SilverFire) [inline framework/validators/Validator.php]: We prefer not to use shorted variables names. Also it makes confusions with `$attributes`

**Comment by SilverFire** [comment]
The code must be formatted according to PSR and Yii2 code style agreements.

**Comment by hscstudio** [comment]
(y)

**Comment by SilverFire** [comment]
Merged with minor adjustments: https://github.com/yiisoft/yii2/commit/da82ff3fb1625597706d9da3c76ca5a0a1ee7ad5

**Comment by SilverFire** [comment]
Thank you!

**Pull request closed without merging**